### PR TITLE
Update status badges in READMEs

### PR DESCRIPTION
**Pull request by michalinacienciala**
After transitioning from version `v1` to `v2` of the system, we changed the
structure of the repository, added new workflows, etc. We need to update README
status badges to show information relevant to `v2`.
We've:
- updated the chat badge to point to the Threshold Network Discord server
- added a docs badge linking to the Threshold Network documentation
- added/updated badges showing the status of the workflows building/testing the
  code in the repo's sub-projects. The displayed status is based on the status
  of the most recently executed workflow related to that project triggered by
  the push to `main`

Refs:
https://github.com/threshold-network/token-dashboard/pull/296
https://github.com/keep-network/coverage-pools/pull/218
https://github.com/keep-network/keep-common/pull/108
https://github.com/keep-network/sortition-pools/pull/194
https://github.com/keep-network/tbtc-v2/pull/420

### CODE REVIEW

File: solidity/ecdsa/README.adoc [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 = Keep ECDSA v2
 
+https://github.com/keep-network/keep-core/.github/workflows/solidity-test.yml[image:https://img.shields.io/github/workflow/status/keep-network/sortition-pools/Test%20Solidity?event=schedule&label=ECDSA%20contracts%20build[Solidity tests status]]
```
Comment (pdyraga) [inline solidity/ecdsa/README.adoc:6]: The `keep-network/sortition-pools/Test` part is probably wrong.

Comment (michalinacienciala) [reply solidity/ecdsa/README.adoc]: Right, fixed in [cfc2721](https://github.com/keep-network/keep-core/pull/3398/commits/cfc27217f3de34591b02564ee04d99e2ed7984ed).

**Comment by michalinacienciala** [comment]
The Slither checks are failing. We need to merge https://github.com/keep-network/keep-core/pull/3432 and then update the `badges` branch with latest `main` code to get this fixed.

**Comment by michalinacienciala** [comment]
https://github.com/keep-network/keep-core/pull/3432 got merged, all the checks are passing now.

**Review by pdyraga — APPROVED** [review]

**Pull request merged**